deepchem/utils/electron_sampler.py

```diff
@@ -80,21 +82,27 @@ def __init__(self,
             Contains the electron's coordinates in a 4D array. The shape of the array should be(batch_no,no_of_electrons,1,3). Can be a 1D empty array, when electron's positions are yet to be initialized.
         steps: int, optional (default 10)
             The number of MCMC steps to be performed when the moves are called.
+        steps_per_update: int (default 10)
+            The number of steps after which the parameters of the MCMC gets updated.
         seed: int, optional (default None)
             Random seed to use.
         symmetric: bool, optional(default True)
             If true, symmetric moves will be used, else asymmetric moves will be followed.
         simultaneous: bool, optional(default True)
             If true, MCMC steps will be performed on all the electrons, else only a single electron gets updated.
+        sampled_electrons: np.ndarray (default np.ndarray([]))

```

Comment:
sample_electrons is a public attribute so it should be documented as such https://numpydoc.readthedocs.io/en/latest/format.html

Reply:
https://numpydoc.readthedocs.io/en/latest/format.html#class-docstring

Reply:
We may want to remove it as an argument since we don't have a good use case right now for user to pass in sampled_electrons as an argument